client-admin/src/components/conversation-admin/comment-moderation/moderate-comments-todo.js

```diff
@@ -41,15 +42,17 @@ class ModerateCommentsTodo extends React.Component {
         />
       )
     })
-    return comments
+    
   }
 
   render() {
+    const max = 100;
```

Comment:
Maybe worth starting to store values like this in a single utils/constants.js file?

Reply:
I'm in support of this for certain config that feels "global" in nature, but this feels rather incidental to me, and maybe better localized with the component.